library/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    // "rootDir": "src",
```

Comment:
I ran into some weird edge case with typescript unable to read JSX expressions where maintainers reported to enable the rootDir.
Due to the fact that `@wessberg/rollup-plugin-ts` is used now in the rollup config instead of babel and typescript plugins separately, it can probably be removed.

I left it commented out, same like the configuration settings for babel and typescript in the rollup config in case you ever need to switch back.

Typescript plugin was unable to produce a flat file tree in /lib, therefore  `@wessberg/rollup-plugin-ts` is used.